lace/test_geometry.py

```diff
@@ -187,3 +187,24 @@ def test_predict_body_units_cm(self):
         mesh = Mesh()
         mesh.v = np.array([[150, 100, 50], [-150, 50, 150]])
         self.assertEqual(mesh.predict_body_units(), 'cm')
+
+    def test_flip(self):
+        raw_box = Mesh(vc('/unittest/serialization/obj/test_box.obj'))
+        box = Mesh(v=raw_box.v, f=raw_box.f)
+        box.reset_normals()
+        original_v = box.v.copy()
+        original_vn = box.vn.copy()
+        original_f = box.f.copy()
+        box.flip(axis=0)
+        box.reset_normals()
+        self.assertEqual(box.f.shape, original_f.shape)
+        self.assertEqual(box.v.shape, original_v.shape)
+        for face, orig_face in zip(box.f, original_f):
+            self.assertNotEqual(list(face), list(orig_face))
```

Comment:
Yeah, the face order issue has been with things like scape templates that were loaded through some other tool and had their faces magically changed without us knowing; this function isn't the place to lock that down... Really what you want is some kind of `canonicalize_faces` function that puts the faces into a standard winding and ordering so they can be easily compared.